packages/blade-coverage-extension/src/index.css

```diff
@@ -3,8 +3,6 @@ body {
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
-  -webkit-font-smoothing: antialiased;
```

Comment:
can we override this for storybook as well? so consumer and storybook looks same?

Reply:
done